components/helper/teamHelper.js

```diff
@@ -0,0 +1,124 @@
+export const lead = [{
+    name : "Aniruddha Basak",
+    image : "https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2FAniruddhaBasak.jpg?alt=media&token=91019834-b7fa-494e-bd57-c940ebd58cf1",
+    linkedin : "https://www.linkedin.com/in/aniruddha2000/",
+    github : ' https://github.com/aniruddha2000/',
+    pos : "DSC Lead"
+},
+{
+    name : "Mr. Subhadip Chandra",
+    image : "https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2Fsubhadipsir.jpeg?alt=media&token=0ffc1ea1-0979-4107-ac03-054637a2d7f9",
+    linkedin : "https://www.linkedin.com/mwlite/in/subhadip-chandra-17b82592",
+    pos : "Faculty Advisor"
+}]
+
+export const coreLeads = [
+    {
```

Comment:
```
Leads
Arkaprabha - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2FarkaprabhaChakraborty.jpeg?alt=media&token=7f3dfa1c-591a-4b1b-a721-d5f48a4a7ab1
Arko - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2FarkoSen.jpeg?alt=media&token=3b565ab0-2976-47e5-9f8d-bf2d72a80c40
Rajarshi - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2FrajarshiChakraborty.jpeg?alt=media&token=fc295aaf-8032-4816-b513-6161be98c1ac
Rohan - https://firebasestorage.googleapis.com/v0/b/dscbppimt-cms-storage.appspot.com/o/member_images%2FrohanRudra.jpeg?alt=media&token=573518b0-f5d2-49f0-9f13-9794e85c1e1e
```

Reply:
Made you the colaborator of DSC interview form where you can take the github link :)